readme.txt

```diff
@@ -1,64 +1,91 @@
 === Jetpack by WordPress.com ===
 Contributors: automattic, adamkheckler, aduth, akirk, allendav, alternatekev, andy, annezazu, apeatling, azaozz, batmoo, barry, beaulebens, blobaugh, cainm, cena, cfinke, chaselivingston, chellycat, csonnek, danielbachhuber, davoraltman, daniloercoli, designsimply, dllh, drawmyface, dsmart, dzver, ebinnion, eliorivero, enej, eoigal, erania-pinnera, ethitter, gcorne, georgestephanis, gibrown, goldsounds, hew, hugobaeta, hypertextranch, iammattthomas, iandunn, jacobshere, jblz, jeherve, jenhooks, jenia, jgs, jkudish, jmdodd, Joen, johnjamesjacoby, jshreve, koke, kraftbj, lamdayap, lancewillett, lschuyler, macmanx, martinremy, matt, matveb, mattwiebe, maverick3x6, mcsf, mdawaffe, michael-arestad, migueluy, mikeyarce, mkaz, nancythanki, nickmomrik, obenland, pento, professor44, rachelsquirrel, rdcoll, ryancowles, richardmuscat, richardmtl, roccotripaldi, samhotchkiss, scarstocea, sdquirk, stephdau, tmoorewp, tyxla, Viper007Bond, westi, yoavf, zinigor
-Tags: WordPress.com, jet pack, comments, contact, gallery, performance, sharing, security, shortcodes, stats, subscriptions, widgets
+Tags: Jetpack, WordPress.com, backup, security, related posts, CDN, speed, anti spam, social sharing, SEO, video, stats
 Stable tag: 4.5
 Requires at least: 4.6
 Tested up to: 4.7
 
-Increase your traffic, view your stats, speed up your site, and protect yourself from hackers with Jetpack.
+The one plugin you need for stats, related posts, search engine optimization, social sharing, protection, backups, faster content, and email lists.
 
 == Description ==
 
-Jetpack simplifies managing WordPress sites by giving you visitor stats, security services, speeding up images, and helping you get more traffic. Jetpack is a free plugin.
+Whatever sort of site you’re running, Jetpack’s features are designed to keep your site secure, increase traffic, and engage your readers.
 
-= Traffic Growth & Insights =
-Jetpack tells you how many visits your site gets and helps you **get more traffic** with tools like Related Posts, Publicize, Enhanced Distribution, and Sharing.
+= Traffic and SEO Tools =
+Traffic is the lifeblood of any website. Jetpack includes:
 
-= Security =
-Jetpack **protects your site** against brute force attacks and unauthorised logins. We also monitor your site for downtime and keep your plugins updated.
+* [free] site stats and analytics
+* [free] automatic promotion and sharing on Facebook, Twitter, LinkedIn, and WordPress.com
+* [free] related posts
+* [paid] search engine optimization tools for Google, Bing, Twitter, Facebook, and WordPress.com
+* [paid] advert programme that includes AdSense, WordAds, Facebook Ads, AOL, Amazon, Google AdX, and Yahoo

```

Comment:
I find this a bit confusing. Does this allow me to add AdSense code to my site, or WordAds if I prefer that?

Since our module aims to make all this easier (you don't need to look for networks, we selected the best for you), should we make that sentence simpler?